src/storage/src/frontend/tests/queries/insert.rs

```diff
@@ -462,25 +463,12 @@ mod constraints {
                     ],
                 )
                 .expect("no system errors"),
-            Err(constraint_violations(
-                ConstraintError::OutOfRange,
-                vec![
-                    vec![
-                        ("column_si".to_owned(), SqlType::SmallInt),
-                        ("column_i".to_owned(), SqlType::Integer)
-                    ],
-                    vec![
-                        ("column_i".to_owned(), SqlType::Integer),
-                        ("column_bi".to_owned(), SqlType::BigInt)
-                    ]
-                ]
-            ))
+            Err(OperationOnTableError::ConstraintViolations(vec![
+                (ConstraintError::OutOfRange, "column_si".to_owned(), SqlType::SmallInt),
+                (ConstraintError::OutOfRange, "column_i".to_owned(), SqlType::Integer),
+                (ConstraintError::OutOfRange, "column_i".to_owned(), SqlType::Integer),
+                (ConstraintError::OutOfRange, "column_bi".to_owned(), SqlType::BigInt),
+            ]))
```

Comment:
I think it is better abort insertion as early as possible because it should abort transaction that will be added in future. So I've created #153 to track enhancement